avogadro/qtplugins/CMakeLists.txt

```diff
@@ -91,64 +94,43 @@ function(avogadro_plugin name description type header pluginClass sources)
 endfunction()
 
 # Now to make the plugins.
-add_subdirectory(3dmol)
 add_subdirectory(bondcentrictool)
 add_subdirectory(bonding)
-add_subdirectory(commandscripts)
 add_subdirectory(coordinateeditor)
 add_subdirectory(copypaste)
 add_subdirectory(crystal)
 add_subdirectory(customelements)
 add_subdirectory(editor)
 add_subdirectory(hydrogens)
-add_subdirectory(importpqr)
-add_subdirectory(lammpsinput)
 add_subdirectory(lineformatinput)
 add_subdirectory(manipulator)
 add_subdirectory(measuretool)
 add_subdirectory(molecularproperties)
 add_subdirectory(navigator)
 add_subdirectory(networkdatabases)
 add_subdirectory(openbabel)
+add_subdirectory(quantumoutput)
```

Comment:
The changes made in this file undo a number of added plugins, this should be edited out, and only change things related to your change.